packages/@react-aria/table/stories/docs-example.css

```diff
@@ -0,0 +1,104 @@
+.aria-table {
+  border-collapse: collapse;
+  width: 300px;
+  height: 200px;
+  display: block;
+  position: relative;
+  overflow: auto;
+
+  .aria-table-rowGroup {
+    display: block;
+  }
+
+  .aria-table-rowGroupHeader {
+    border-bottom: 2px solid var(--spectrum-global-color-gray-800);
+    position: sticky;
+    top: 0;
+    background: var(--spectrum-gray-100);
+    width: fit-content;
+  }
+
+  .aria-table-rowGroupBody {
+    max-height: 200px;
+  }
+
+  .aria-table-row {
+    display: flex;
+  }
+
+  .aria-table-headerCell {
+    padding: 5px 10px;
+    outline: none;
+    cursor: default;
+    display: block;
+    flex: 0 0 auto;
+    box-sizing: border-box;
+    text-align: left;
+
+    .aria-table-headerTitle {
+      width: 100%;
+      text-align: left;
+      border: none;
+      background: transparent;
+      flex: 1 1 auto;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      margin-inline-start: -6px;
+      outline: none;
+
+      &.focus {
+        outline: 2px solid orange;
+      }
+    }
+  }
+
+  .aria-table-resizer {
+    width: 6px;
+    background-color: grey;
+    cursor: col-resize;
+    height: auto;
+    touch-action: none;
+    flex: 0 0 auto;
+    box-sizing: border-box;
+    border: 2px;
+    border-style: none solid;
+    border-color: transparent;
+    background-clip: content-box;
+
+    &.focus {
+      background-color: orange;
+    }
+
+    &.resizing {
+      border-color: orange;
+      background-color: transparent;
+    }
+
+    input {
+      margin: 0px;
+    }
```

Comment:
Needed to fix the following:
1. User uses arrow right to move from the column header to the resizer
2. Focus lands on the visually hidden input due to useGridCell (aka moving between focusable children in a cell)
3. Calls scrollIntoViewport causing a the table to get centered in viewport

Kinda gross, but needed to override the browser's default 2px margin applied to a range type input. That margin was causing `scrollIntoView({block: 'nearest'})` calls to shift the visually hidden input a little bit -> makes scrollIntoViewport think that we need to scroll the entire table into view. 

Tried making the resizer div itself focusable, but the visually hidden input is still going to be focusable and thus useGridCell would have three elements to cycle through (column header text -> resizer div -> visually hidden range input) where ideally we'd only have the column header and the resizer div as focusable elements in this case. We can't make the visually hidden input non-focusable either since we need to programmatically focus it for resize operations.

Reply:
I was a bit apprehensive about providing input styles from the useTableColumnResize hook since it felt too prescriptive and we usually aren't in the business of providing styles from the hooks. However, we do state that the input is/should be visually hidden (thus low chance a user would modify the actual look of the input element) and we have precedence of styles being provided from the color hooks so I suppose I'll go that route